src/Symfony/Component/DependencyInjection/Tests/Compiler/ResolveChildDefinitionsPassTest.php

```diff
@@ -396,4 +396,20 @@ protected function process(ContainerBuilder $container)
         $pass = new ResolveChildDefinitionsPass();
         $pass->process($container);
     }
+
+    /**
+     * @expectedException \Symfony\Component\DependencyInjection\Exception\ServiceCircularReferenceException
```

Comment:
could be worth adding an `@expectedExceptionMessage`: right now, I get: `Service "a": Service "a": Service "a": Service "a": Circular reference detected for service "c", path: "c -> b -> a -> c".`, which might need some tweaks :)

Reply:
hmm, that's `resolveDefinition` prefixing the exception message. And the multiple prefixing looks really weird though.

Reply:
Oups, didn't see that. I just pushed a fix.

Reply:
still worth adding `@expectedExceptionMessage` :)

Reply:
Maybe `@expectedExceptionMessageRegExp` to test if exception message is not prefixed?